va/va_enc_av1.h

```diff
@@ -670,7 +672,6 @@ typedef struct  _VAEncPictureParameterBufferAV1 {
      *  0: 16x16 block size, default value;
      *  1: 32x32 block size;
      *  2: 64x64 block size;
-     *  3: 8x8 block size.
```

Comment:
Why is this removed? 
For the consisency to vp9, should this be VA_SEGID_BLOCKXXX?

Reply:
updated.